### PR TITLE
Feature/design ux fixes

### PR DESCRIPTION
**Pull request by LordFren**

### CODE REVIEW

File: src/static/css/component/_content.less [hunk 1]
```diff
@@ -0,0 +1,7 @@
+h1, h2, h3, h4, h5{
```
Comment (ttaylorr) [inline src/static/css/component/_content.less]: make sure there is a space after `h5` and the `{`

File: src/static/css/component/_content.less [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	margin-top: 40px;
+}
+h1, h2{
+	color: #22517d;
+	font-weight: 500;
+}
```
Comment (ttaylorr) [inline src/static/css/component/_content.less]: make sure there is a newline at the end of the file.

File: src/template/api/chatproto.jade [hunk 3]
```diff
@@ -6,7 +6,7 @@ block content
         .container
             .col-sm-3: h1 Chat protcol
             .col-sm-9
-                p Information for connecting to, interacting with, and decoding chat server information,
+                p Information for connecting to, interacting with, and decoding chat server information
```
Comment (ttaylorr) [inline src/template/api/chatproto.jade]: we can probably replace this with a period.

File: src/template/layout/master.jade [hunk 4]
```diff
@@ -34,8 +34,8 @@ html
                     ul.footer-list
                         li: a(href='#{beamUrl}/legal/tos') Terms of Service
                         li: a(href='#{beamUrl}/legal/privacy') Privacy Policy
-                        li: a(href='#{beamUrl}/contact') Contact
-                        li: a(href='#{beamUrl}') Beam
+                        li: a(href='https://beam.pro/contact/contact') Contact
```
Comment (ttaylorr) [inline src/template/layout/master.jade]: why did this change?

Comment (njb-said) [inline src/template/layout/master.jade]: @ttaylorr it was undefined

**Pull request closed without merging**